### ISSUE TITLE
g728float: tests fail

### ISSUE THREAD

**Issue by dennisguse** [openitu/STL#53]
Failing  tests:
* g728-enc5-verify
* g728-dec1-verify
* g728-dec2-verify
* g728-dec3-verify
* g728-dec4-verify
* g728-dec5-verify
* g728-dec6-verify
* g728-dec-post-verify

Files are somehow different.

Used Ubuntu 16.10 (64bit).
Patch status today.

[LastTest.log.txt](https://github.com/dennisguse/stl2009/files/992258/LastTest.log.txt)

**Comment by ludomal** [comment]
```src/g728/g728float/CMakeLists.txt``` and ```src/g728/g728fixed/CMakeLists.txt``` were updated to fix these issues.

* ```g728-enc5``` test was not part of the original STL2009. The vector is not available so I removed that test.

* ```g728-dec-post-verify``` test was passing ```enc``` instead of ```dec``` to the encoder.

* ```g728-dec?-verify``` : the original makefile was setting ```USE_DOUBLES``` pre-processor for compiling g728. Without setting this preprocessor floats instead of doubles were used, thus the failed tests.
